.travis.yml

```diff
@@ -7,8 +7,9 @@ env:
   - TOXENV=quality
   - TOXENV=docs
   - TOXENV=django22-drf312
-  - TOXENV=django22-drflatest
```

Comment:
why aren't we testing Django 2.2 with the latest version of DRF?

Reply:
travis moderniser made this change